tests/test-cases/Group1-Docker-Commands/1-41-Docker-Commit.robot

```diff
@@ -0,0 +1,72 @@
+Commit nano to image
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --name commit1 ${debian} tail -f /dev/null
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} exec commit1 apt-get update
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} exec commit1 apt-get install nano
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop -t1 commit1
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} commmit commit1 debian-nano
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run debian-nano whereis nano
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  /usr/bin/nano
+
+Commit env variable to image
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --name commit2 ${debian} tail -f /dev/null
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop -t1 commit2
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} commit -c "ENV TEST commitEnvTest" commit2 debian-env
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run debian-env env
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  TEST\=commitEnvTest
+
+Unsupported commit command
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --name commit3 ${debian} tail -f /dev/null
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop -t1 commit3
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} commit -c "RUN apt-get install nginx" commit3 debian-env
+    Should Not Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  Error response from daemon: run is not a valid change command
+
+Commit with author and message
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --name commit4 ${debian} tail -f /dev/null
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop -t1 commit4
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} commit -a Robot -m "Robot made a commit" -c "ENV TEST commitEnvTest" commit4 debian-auth
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} inspect debian-auth
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  "Author" : "Robot"
```

Comment:
should be "Author": "Robot", "Comment": "Robot made a commit"
No space after the key